src/main/java/com/mraof/minestuck/block/MSBlocks.java

```diff
@@ -210,51 +299,90 @@ private static Block coalOre(BlockBehaviour.Properties properties)
+	public static final RegistryObject<WallBlock> POLISHED_BLACK_STONE_WALL = REGISTER.register("polished_black_stone_wall", () -> new WallBlock(copy(POLISHED_BLACK_STONE.get())));
 	
 	public static final RegistryObject<Block> BLACK_STONE_BRICKS = REGISTER.register("black_stone_bricks", () -> new Block(copy(BLACK_STONE.get())));
 	public static final RegistryObject<StairBlock> BLACK_STONE_BRICK_STAIRS = REGISTER.register("black_stone_brick_stairs", () -> new StairBlock(() -> MSBlocks.BLACK_STONE_BRICKS.get().defaultBlockState(), copy(BLACK_STONE_BRICKS.get())));
 	public static final RegistryObject<SlabBlock> BLACK_STONE_BRICK_SLAB = REGISTER.register("black_stone_brick_slab", () -> new SlabBlock(copy(BLACK_STONE_BRICKS.get())));
+	public static final RegistryObject<WallBlock> BLACK_STONE_BRICK_WALL = REGISTER.register("black_stone_brick_wall", () -> new WallBlock(copy(BLACK_STONE_BRICKS.get())));
 	
 	public static final RegistryObject<Block> BLACK_STONE_COLUMN = REGISTER.register("black_stone_column", () -> new MSDirectionalBlock(copy(BLACK_STONE.get())));
 	public static final RegistryObject<Block> CHISELED_BLACK_STONE_BRICKS = REGISTER.register("chiseled_black_stone_bricks", () -> new Block(copy(BLACK_STONE_BRICKS.get())));
 	public static final RegistryObject<Block> CRACKED_BLACK_STONE_BRICKS = REGISTER.register("cracked_black_stone_bricks", () -> new Block(copy(BLACK_STONE_BRICKS.get())));
 	
 	//Flowery
 	public static final RegistryObject<Block> FLOWERY_MOSSY_COBBLESTONE = REGISTER.register("flowery_mossy_cobblestone", () -> new Block(Block.Properties.of().mapColor(MapColor.COLOR_GRAY).instrument(NoteBlockInstrument.BASEDRUM).requiresCorrectToolForDrops().strength(1.5F, 6.0F)));
+	public static final RegistryObject<StairBlock> FLOWERY_MOSSY_COBBLESTONE_STAIRS = REGISTER.register("flowery_mossy_cobblestone_stairs", () -> new StairBlock(FLOWERY_MOSSY_COBBLESTONE.get().defaultBlockState(), copy(FLOWERY_MOSSY_COBBLESTONE.get())));
+	public static final RegistryObject<SlabBlock> FLOWERY_MOSSY_COBBLESTONE_SLAB = REGISTER.register("flowery_mossy_cobblestone_slab", () -> new SlabBlock(copy(FLOWERY_MOSSY_COBBLESTONE.get())));
+	public static final RegistryObject<WallBlock> FLOWERY_MOSSY_COBBLESTONE_WALL = REGISTER.register("flowery_mossy_cobblestone_wall", () -> new WallBlock(copy(FLOWERY_MOSSY_COBBLESTONE.get())));
 	
 	public static final RegistryObject<Block> FLOWERY_MOSSY_STONE_BRICKS = REGISTER.register("flowery_mossy_stone_bricks", () -> new Block(Block.Properties.of().mapColor(MapColor.COLOR_GRAY).instrument(NoteBlockInstrument.BASEDRUM).requiresCorrectToolForDrops().strength(1.5F, 6.0F)));
-	public static final RegistryObject<StairBlock> FLOWERY_MOSSY_STONE_BRICK_STAIRS = REGISTER.register("flowery_mossy_stone_brick_stairs", () -> new StairBlock(() -> MSBlocks.FLOWERY_MOSSY_STONE_BRICKS.get().defaultBlockState(), copy(FLOWERY_MOSSY_STONE_BRICKS.get())));
```

Comment:
This removal seems a bit odd. With the non-supplier constructor being deprecated, I think it's better to keep to the supplier-based constructor instead.

Reply:
This will depend on whether I'm adding or removing these, I'll revert this if I'm actually adding them.

Reply:
Not sure I follow. Is there any undecided question that you're referring to?

Reply:
The previously commented-out lines, but disregard. It's been added back.

Reply:
Alright, thanks!
I'd say that the same thing should apply to the new stairs blocks as well. You should be able to spot the deprecations directly in the file in your IDE, or else you could do a search on the deprecated constructor in your IDE.